recipe/build_base.sh

```diff
@@ -8,9 +7,14 @@ export AR=$GCC_AR
 export MESON_ARGS="${MESON_ARGS} --pkg-config-path=${PREFIX}/lib/pkgconfig"
 export MESON_ARGS="${MESON_ARGS} -Dsystem-freetype=true -Dsystem-qhull=true"
 
-mkdir builddir
-$PYTHON -m mesonbuild.mesonmain setup builddir $MESON_ARGS
-cat builddir/meson-logs/meson-log.txt
-$PYTHON -m build --wheel \
-        --no-isolation --skip-dependency-check -Cbuilddir=builddir
+# meson-python already sets up a -Dbuildtype=release argument to meson, so
+# we need to strip --buildtype out of MESON_ARGS or fail due to redundancy
+MESON_ARGS_REDUCED="$(echo $MESON_ARGS | sed 's/--buildtype release //g')"
```

Comment:
Do we have an issue tracking removing this flag from `MESON_ARGS` upstream?

Reply:
No, it's their default. Our compilers need to change, and they did as of [clang 17](https://github.com/conda-forge/clang-compiler-activation-feedstock/pull/112) / [GCC 13](https://github.com/conda-forge/clang-compiler-activation-feedstock/pull/112)

Reply:
Sorry I meant raise a conda-forge issue, but it sounds like that already was fixed. Do you have a link?

Reply:
There are two links in my comment...? 🤔

Reply:
Oh oops. Sorry was on my phone, which didn't make this clear. Thanks!